client/dist/css/related_items.css

```diff
@@ -9,7 +9,7 @@
 .RelatedItemCard, .OutfitListCard {
     background: rgb(235, 233, 233);
```

Comment:
I love your CSS style, especially the hover animation. I personally would add object-fit: cover in CSS to the image so that the image stays in its original scales and won't stretch.